<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1528</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1521</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(170, 0, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Proje Bilgileri</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>931</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>461</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Metraj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Assembly</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>641</width>
        <height>211</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>751</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>37</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Assembly Pozu : </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>80</y>
        <width>841</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Single</string>
     </attribute>
     <widget class="QTableView" name="tableView_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>931</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Platinalar</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>610</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1528</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionA"/>
   </widget>
   <widget class="QMenu" name="menuAyarlar">
    <property name="title">
     <string>Ayarlar</string>
    </property>
    <addaction name="actionMaliyet_ayarlar"/>
    <addaction name="actionGenel_ayarlar"/>
   </widget>
   <widget class="QMenu" name="menuRaporlar">
    <property name="title">
     <string>Raporlar</string>
    </property>
    <addaction name="actionKaynak_raporlar"/>
    <addaction name="actionat_m_Raporlar"/>
    <addaction name="actionBoya_raporlar"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuAyarlar"/>
   <addaction name="menuRaporlar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionA">
   <property name="text">
    <string>Aç...</string>
   </property>
  </action>
  <action name="actionMaliyet_ayarlar">
   <property name="text">
    <string>Maliyet ayarları</string>
   </property>
  </action>
  <action name="actionGenel_ayarlar">
   <property name="text">
    <string>Genel ayarlar</string>
   </property>
  </action>
  <action name="actionKaynak_raporlar">
   <property name="text">
    <string>Kaynak raporları</string>
   </property>
  </action>
  <action name="actionat_m_Raporlar">
   <property name="text">
    <string>Çatım raporları</string>
   </property>
  </action>
  <action name="actionBoya_raporlar">
   <property name="text">
    <string>Boya raporları</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
